scripts/py_matter_yamltests/matter_yamltests/definitions.py

```diff
@@ -232,14 +242,29 @@ def __get_by_id(self, cluster_id: int, target_id: int, target_type: str):
 
         return targets.get(target_id)
 
-    def __enforce_casing(self, target_name: str, targets: list):
-        if targets.get(target_name) is not None:
-            return
+    def __get_targets_by_cluster_name(self, cluster_name: str, target_type: _ItemType):
+        if not cluster_name:
+            return None
+
+        # The idl parser remove spaces
```

Comment:
Again, are spaces all that gets removed?

Reply:
See answer at https://github.com/project-chip/connectedhomeip/pull/25109#discussion_r1108811896